### ISSUE TITLE
Make _updateCollectionData null-safe

### ISSUE THREAD

**Issue by sionzee** [dmapper/react-sharedb#4]
When you scope your data eg.
`let $scope = model.scope('_page.variable')
`And set it to null
`$scope.set(null);
`
In method react-sharedb/lib/subscribe.js: _updateCollectionData, will be called:
`this.data = (0, _observerUtil.observable)(this.data);
`
Where this.data == null;
In observer methods it will test if it is not instrument and call on it:
`var constructor = ref.constructor
`Where ref will be null.

Temporary fix for this is to set false insteadof null.

**Comment by cray0000** [comment]
good catch. Though it's better to use model.del() for such operations. I'll take a look at how to patch it.
